SAMtemplates/load_balancer/main.yaml

```diff
@@ -0,0 +1,155 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: |
+  Main template for Load Balancer with all resources included, supporting mutual TLS
+
+Parameters:
+  StackName:
+    Type: String
+
+  EnableMutualTLS:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+
+  TruststoreVersion:
+    Type: String
+
+  TruststoreFile:
+    Type: String
+
+  VersionNumber:
+    Type: String
+
+  CommitId:
+    Type: String
+
+Conditions:
+  CreateTruststore: !Equals [!Ref EnableMutualTLS, true]
+
+Resources:
+  # Create Security Group for the Load Balancer
+  LoadBalancerSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId: !ImportValue vpc-resources:VpcId
+      GroupDescription: Allow inbound HTTPS traffic
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-lb-sg"
```

Comment:
this should sub in the StackName parameter rather than the AWS one

Reply:
The value has been changed to `!Sub "${StackName}-lb-sg"`